src/store/modules/DownloadModule.ts

```diff
@@ -50,11 +52,11 @@ export const DownloadModule = {
     }),
 
     actions: <ActionTree<State, RootState>>{
-        addDownload({state}, initialMods: string[]): UUID {
+        addDownload({state}, combos: ThunderstoreCombo[]): UUID {
```

Comment:
After the latest refactors it seems that having the Vue components call this action is unnecessary, as they only passes the received download ID back to `downloadAndInstallCombos`. Vue could simply call `downloadAndInstallCombos` which instead would call this action internally. We could even consider prefixing the name of this action with `_` to indicate it's not intended to be called directly.

Reply:
I changed it so that addDownload is only called internally and prefixed it with an underscore.
I also added the underscore to other similar functions.